LobbyServer2/LobbyServer/LobbyServerProtocol.cs

```diff
@@ -537,6 +549,13 @@ protected void SetContextualReadyState(ContextualReadyState contextualReadyState
             }
             else
             {
+                if (CurrentServer != null)
+                {
+                    if (contextualReadyState.ReadyState == ReadyState.Ready)
+                    {
+                        CurrentServer.GetPlayerInfo(AccountId).ReadyState = ReadyState.Ready;
```

Comment:
Do we not need to reset it if client unreadies?

Reply:
No once client ready up they can't unready at that state